geode-core/src/main/java/org/apache/geode/management/internal/api/LocatorClusterManagementService.java

```diff
@@ -155,23 +159,33 @@ public ClusterManagementResult create(CacheElement config) {
 
     // add the config object which includes the HATOS information of the element created
     if (result.isSuccessful()) {
-      result.setResult(Collections.singletonList(config));
+      ClusterManagementResult<R> ret = get(config);
```

Comment:
I think you are trying to do too much in this PR. Let's first sort out the linkage between the static and runtime element so that the `list` method doesn't need to take a class parameter. let's first not change what's returned in the "create". We can deal with that on another PR.

Reply:
I'll see if it can be separated.  I wanted to keep a clean line between input and output types, but maybe there is another way.

Reply:
I would still prefer that user would simply get a URI and it's up to them to decide whether they want to make another round trip or not.

Reply:
I think that would be a great change for another PR.  For this one, I'm going work on paring it down to the minimum changeset (including backing out this behavior change to `create`)

Reply:
reverted these changes to `create`